security/cmd/istio_ca/main.go

```diff
@@ -97,6 +108,113 @@ var (
 	}
 )
 
+// LivenessCheckController updates the availability of the liveness probe of the CA instance
+type LivenessCheckController struct {
```

Comment:
Could you move this LivenessCheckController to a separate file? You can create a directory: security/pkg/probe/.

Reply:
Done